src/data-access/repositories/test/mongo-test-database.ts

```diff
@@ -11,12 +11,12 @@ export class MongoTestDatabase {
     jest.setTimeout(15000)
   }
 
-  public async setupDatabase(): Promise<void> {
+  public async setupDatabaseConnection(): Promise<void> {
```

Comment:
If I were to call this method, I would expect that only the database connection is established – not that the entire database server is created.

Reply:
Renamed to `setupDatabaseServer`, and similarly for the teardown method.